com.maplerain.telsis_translator_flutter.yml

```diff
@@ -0,0 +1,38 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/main/data/flatpak-manifest.schema.json
+
+---
+app-id: com.maplerain.telsis_translator_flutter
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: telsis_translator_flutter
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+modules:
+  # telsis_translator_flutter
+  - name: telsis_translator_flutter
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - chmod +x telsis_translator_flutter
+      - mv com.maplerain.telsis_translator_flutter.png /app/share/icons/hicolor/256x256/apps/
+      - mv com.maplerain.telsis_translator_flutter.desktop /app/share/applications/
+      - mv com.maplerain.telsis_translator_flutter.metainfo.xml /app/share/metainfo/
+      - mv * /app/
+      - mkdir -p /app/bin
+      - ln -s /app/telsis_translator_flutter /app/bin/telsis_translator_flutter
```

Comment:
```suggestion
      - install -Dm644 com.maplerain.telsis_translator_flutter.png /app/share/icons/hicolor/256x256/apps/
      - install -Dm644 com.maplerain.telsis_translator_flutter.desktop /app/share/applications/
      - install -Dm644 com.maplerain.telsis_translator_flutter.metainfo.xml /app/share/metainfo/
      - mv data lib telsis_translator_flutter /app/
      - mkdir -p /app/bin
      - ln -s /app/telsis_translator_flutter /app/bin/telsis_translator_flutter
```

ok, that's 6 lines.

Reply:
Thank you for the suggestion. I have changed the archive to remove the `packaging\linux` directory (also updated the sha256sum) and made the relevant changes. Please take a look again.